### DISCUSSION TITLE
No BG in Nightscout follower

### DISCUSSION THREAD

**Discussion by psonnera** [NightscoutFoundation/xDrip#2699] [Q&A]
AAPS dev uploading BG and treatments in NS dev (15).
xDrip+ setup as a Nightscout follower. Treatments download but not BG.
![image](https://user-images.githubusercontent.com/33809361/221237848-73bf52c5-fe7f-4b36-9336-3c064238f900.png)
![image](https://user-images.githubusercontent.com/33809361/221238005-b8a6fcce-8f29-4aa9-9b6e-b23f039916e3.png)
![image](https://user-images.githubusercontent.com/33809361/221238044-72c7b3a8-ea2b-4294-9a31-63c96d6ebdc4.png)
![image](https://user-images.githubusercontent.com/33809361/221238075-e1bd00af-005c-4906-9eb7-1556487b74fe.png)

**Comment by claramiao** [comment]
Hi, im having the same problem. Anyone know how to solve ir? Thx

**Comment by Navid200** [comment]
What triggered this?
The update of Nightscout to 15.0.0?

**Comment by kristinsmotz** [comment]
This is my problem, copied from Facebook. Had a sensor error, followed by sensor change, and never got back signal in xdrip.

**Comment by kristinsmotz** [comment]
![Screenshot_20230224_222613_xDrip+](https://user-images.githubusercontent.com/71758682/221306994-28d456a7-558a-414a-9363-3a2aed13f98b.jpg)
I did get one random bg value tonight, then back to signal loss.

**Reply by Navid200** [reply]
Please don't add a link to an image.  If you want to add a screenshot, just click on the bottom border of your post while editing it and it will let you choose the file you want to attach.

**Reply by kristinsmotz** [reply]
> Please don't add a link to an image. If you want to add a screenshot, just click on the bottom border of your post while editing it and it will let you choose the file you want to attach.

That's what I did but I guess I was too impatient to let it finish uploading before submitting.

**Comment by kristinsmotz** [comment]
Another random bg value came in this morning. 
![Screenshot_20230225_085841_xDrip+](https://user-images.githubusercontent.com/71758682/221348488-85cae05e-de6e-47d2-aa44-4c2248c72e8a.jpg)

**Comment by claramiao** [comment]
hello, any idea what it could be? I'm having exactly the same problem...

I'm using NS 15 (dev)

I have already updated the xdrip follower app to the latest version 02/23 but still not receiving the BG, only the treatments. on the nightscout website, everything is ok, BG and treatments.

**Comment by Navid200** [comment]
I don't think we are going to make any progress if you guys keep reporting the symptoms.
I think it is clear that you are experiencnig this problem.  No one is disputing that.

I asked a question and none has bothered answering.
I ask again:

What has triggered this?
For example, have you experienced this after upgrading to Nightscout 15.0.0?

**Reply by kristinsmotz** [reply]
As I mentioned above, trigger for me was a sensor error then no signal came back after we switched sensor. Updated nightscout to 15 and AAPS to dev-h over a week ago and had signal in all apps throughout the week until our second round of sensor error followed by sensor change, where only xdrip lost bg's, while they are still visible in nightscout, aaps, AAPSClient, and on fitbit versa watches fetching from  ns.

**Reply by Navid200** [reply]
OK, thanks.

If you change the URL of Nightscout follower in xDrip to something else and apply.  Then, change it back, will that resolve the problem?

**Reply by kristinsmotz** [reply]
No, that doesn't solve it, I've tried that as well. Even tried deleting cache and data and setting up xdrip from scratch. I get some random bg's in with several hours between, but consistently get treatment records like carbs, bolus, smb, basal change, notes.

**Reply by claramiao** [reply]
> I don't think we are going to make any progress if you guys keep reporting the symptoms. I think it is clear that you are experiencnig this problem. No one is disputing that.
> 
> I asked a question and none has bothered answering. I ask again:
> 
> What has triggered this? For example, have you experienced this after upgrading to Nightscout 15.0.0?

hello, sorry about that. yes, what caused this problem was the update to NS 15. as soon as I updated, everything worked normally.

I did the update on 02/23, +-24h later he stopped receiving the BG in the xdrip followers.

**Reply by claramiao** [reply]
Here some screenshots after clear xdrip data then configure as a follower

I tried clearing the data and even reinstalling xdrip. as soon as I put the url as us follower, it loads an amount of data from the previous BG and doesn't receive any more, it only receives the treatments.

**Comment by claramiao** [comment]
![Screenshot_20230225_121615_xDrip+](https://user-images.githubusercontent.com/50266247/221367977-33df478f-4f0f-40e6-952b-bdac7d2b3022.jpg)
![Screenshot_20230225_115148_xDrip+](https://user-images.githubusercontent.com/50266247/221367979-d3b6725c-0c3a-4063-8144-ed54ec451422.jpg)
![Screenshot_20230225_115132_xDrip+](https://user-images.githubusercontent.com/50266247/221367982-e4c5a2b8-dd37-4e80-a358-938c2b6fa1e5.jpg)

**Comment by Navid200** [comment]
Can any of you downgrade Nightscout to 14.2.6 and see if the problem persists or not?

**Comment by claramiao** [comment]
Hello guys, my xdrip started working again last night. I didn't do anything to make this happen.. it's working fine so far..

**Reply by kristinsmotz** [reply]
Mine did too, only thing I tried last night was switching from beta to stable in automatic update settings.

**Reply by jamorham** [reply]
This doesn't really make any sense so something else must have been going on.

**Reply by kristinsmotz** [reply]
Someone posted this link on my thread in the Facebook group: https://github.com/nightscout/cgm-remote-monitor/issues/7910?fbclid=IwAR1xhTjLNAgx3vflLKoYWC8Fd5gqDTfT1bm_nIJ9pCFz1jKfO_OtHshbE8U
Could have possibly been the same issue, I don't know. I still have consistent signals now since they came back up sometime between Saturday night and Sunday morning.
